Dockerfile

```diff
@@ -1,16 +1,46 @@
-FROM 		phusion/baseimage
-MAINTAINER 	Casey Bisson <casey.bisson@gmail.com
+FROM alpine:3.2
+MAINTAINER 	Casey Bisson <casey.bisson@gmail.com>
 
-# Update Apt
-RUN apt-get update
+# Alpine packages
+RUN apk --update \
+    add \
+        jq \
+        curl \
+        bash \
+        ca-certificates && \
+    curl -Ls https://circle-artifacts.com/gh/andyshinn/alpine-pkg-glibc/6/artifacts/0/home/ubuntu/alpine-pkg-glibc/packages/x86_64/glibc-2.21-r2.apk > /tmp/glibc-2.21-r2.apk && \
```

Comment:
@tgross This line `apk del go git gcc musl-dev make bash && \` mitigates the size of the container because we remove all of the dependencies within the same build step.

@misterbisson Yes, those are great ideas for a future version. Carry on with the good work. :+1: